File: _branchPedro/Back/src/ProEventos.Domain/Biblioteca/Acervo.cs
using System;

namespace ProEventos.Domain.Biblioteca
{
    public class Acervo
    {
        public int Id { get; set; }
        public string? Titulo { get; set; }
        public string? Subtitulo { get; set; }
        public string? Categoria { get; set; }
        public int? ISBN { get; set; }
        public string? Resumo { get; set; }
        public DateTime? AnoPublicacao { get; set; }
        public int? QuantidadeDisponivel { get; set; }
        public int? QuantidadeTransito { get; set; }
        public int? QuantidadeAlocada { get; set; }
        public string? Capa { get; set; }
        public int? PatrimonioId { get; set; }
        public Patrimonio? Patrimonio { get; set; }
    }
}

File: src/BibliotecaCorporativa/frontend/BibCorp-app/src/app/acervos/models/acervo/Acervo.ts
import { Patrimonio } from "src/app/patrimonios";

export class Acervo {
  id: number;
  isbn: string;
  genero: string;
  titulo: string;
  autor: string;
  subTitulo: string;
  resumo: string;
  anoPublicacao: string;
  dataCriacao: string;
  editora: string;
  edicao: string;
  capaUrl: string;
  qtdPaginas: number;
  comentarios: string;
  qtdeDisponivel: number;
  qtdeEmTransito: number;
  qtdeEmprestada: number;
  patrimonios: Patrimonio[];
}
